lib/guardian.rb

```diff
@@ -434,9 +434,8 @@ def can_send_private_messages?(notify_moderators: false)
     authenticated? &&
       # User can send PMs, this can be covered by trust levels as well via AUTO_GROUPS
       (
-        is_staff? || from_bot || from_system ||
-          (@user.in_any_groups?(SiteSetting.personal_message_enabled_groups_map)) ||
-          notify_moderators
+        from_bot || from_system || notify_moderators ||
+          @user.in_any_groups?(SiteSetting.personal_message_enabled_groups_map)
```

Comment:
Removed `is_staff?` and moved `notify_moderators` condition because they can all fit on one line without awkward formatting. 😅